gsy_framework/sim_results/kpi.py

```diff
@@ -236,7 +237,7 @@ def to_dict(self):
         return {"base_case_cost": self.base_case_cost,
                 "utility_bill": self.utility_bill,
                 "fit_revenue": self.fit_revenue,
-                "d3a_cost": self.d3a_cost,
+                "gsy_e_cost": self.gsy_e_cost,
```

Comment:
Please ping Rafael to change that once it is merged, this will make the frontend fail.

Reply:
Yes, will do